CHANGES.md

```diff
@@ -83,6 +85,7 @@ To be released.
  -  Added `Swarm<T>.TrustedAppProtocolVersionSigners` property.
     [[#266], [#815]]
  -  Added `Peer.IsCompatibleWith()` method.  [[#266], [#815]]
+ -  Added `InvalidTxByBlockPolicyException` class.  [[#827]]
```

Comment:
How about adding a new term for this case which is distinct from “invalid tx” instead of calling both cases “invalid tx”?

Reply:
I agree. Let's think about the more proper term.

Reply:
I changed to use the term <q>follow policy</q> and <q>policy violation</q>. Please take a look.